src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,47 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * Class for user interaction.
+ */
+public class Ui {
+    private Scanner sc;
```

Comment:
Since this scanner has a relatively long scope, it might be better to give it a longer, more descriptive name.